### ISSUE TITLE
cannot pull - tx ERROR: not enough values to unpack (expected 2, got 1)

### ISSUE THREAD

**Issue by evgenyfadeev** [transifex/transifex-client#346]
Hello, when I run tx pull or tx status, I'm getting the error "tx ERROR: not enough values to unpack (expected 2, got 1)"

`tx --version`: 0.14.4, py 3.9, x86_64

Here is my .transifexrc:

```
[https://www.transifex.com]
api_hostname = https://api.transifex.com
hostname = https://www.transifex.com
password = <api token>
rest_hostname = https://rest.api.transifex.com
token = <api token>
username = api
```

.tx/config:

```
[main]
host = https://www.transifex.com

[o:askbot:p:askbot:r:askbot-translation-part1-main]
file_filter = askbot/locale/<lang>/LC_MESSAGES/django.po
source_file = askbot/locale/en/LC_MESSAGES/django.po
source_lang = en
type = PO
resource_name = askbot-translation-part1-main

[o:askbot:p:askbot:r:askbot-translation-part2-js]
file_filter = askbot/locale/<lang>/LC_MESSAGES/djangojs.po
source_file = askbot/locale/en/LC_MESSAGES/djangojs.po
source_lang = en
type = PO
resource_name = askbot-translation-part2-js
```

**Comment by evgenyfadeev** [comment]
Haha, now I get it - this one is deprecated.

Perhaps it's a good idea to put scary depecation emojis on the README and on the Pipy page - like a big stop sign.

Even thought the readme text does mention the new version, the title is "The New Client 🎉" - does not give an impression that this code should no longer be used.

**Comment by keithbowes** [comment]
Yeah, during the deprecation period, tx would show deprecation warnings every time you tried to do something and then one day, it stopped working entirely and I had to migrate to the new client.  It's unfortunate that no one has forked this and updated it to the new API, for those who want to continue using the Python version (though, from a user's perspective, there's very little difference between the Python and Go versions, but as you say, getting it from Pipy no longer works, and a lot of OSes still have the old version in their repos).
